image_client/botany_importer.py

```diff
@@ -54,7 +54,6 @@ def process_barcode(self, barcode, filepath_list):
         self.logger.debug(f"Barcode: {barcode}")
         sql = f'''select CollectionObjectID from collectionobject where CatalogNumber={barcode};'''
         collection_object_id = self.specify_db_connection.get_one_record(sql)
-        print(collection_object_id)
         force_redacted = False
```

Comment:
I like to keep my old "prints" and just make them logger.debugs, but this one is indeed pretty minimal. case-by-case basis.